### PR TITLE
Task/rfr 57 fix case sensitive login

### PR DESCRIPTION
**Pull request by hb0**
This PR ensures the login attempts don't fail due to case-sensitive usernames (usually an email address).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -97,7 +97,7 @@ ext {
   slf4jVersion = '1.7.29'
   commonsLangVersion = '3.12.0'
   logbackVersion = '1.2.5'
-  cyfaceApiVersion = '1.0.0'
+  cyfaceApiVersion = '2.0.1-beta1'
```
Comment (hb0) [inline build.gradle:100]: TODO

**Review by muthenberg — CHANGES_REQUESTED** [review]
Since I already completely rewrote this code, I would appreciate if you could reduce the changes to the bare necessary minimum, since all of these will result in conflicts I need to resolve. So please do not rename any variables or change methode visibilities and the like.

**Comment by hb0** [comment]
> Since I already completely rewrote this code, I would appreciate if you could reduce the changes to the bare necessary minimum, since all of these will result in conflicts I need to resolve. So please do not rename any variables or change methode visibilities and the like.

I undid the unnecessary changes

File: src/main/kotlin/de/cyface/collector/handler/MeasurementHandler.kt [hunk 2]
```diff
@@ -123,7 +119,7 @@ class MeasurementHandler(
             // Handle first chunk
             val contentRange = contentRange(request, bodySize)
             if (session.get<Any?>(UPLOAD_PATH_FIELD) == null) {
-                handleFirstChunkUpload(ctx, request, session, loggedInUser, contentRange, metaData, null)
```
Comment (muthenberg) [inline src/main/kotlin/de/cyface/collector/handler/MeasurementHandler.kt:126]: Please keep loggedInUser here. I already changed this in my pending commit.

**Review by muthenberg — CHANGES_REQUESTED** [review]
see my comment

**Pull request merged**